src/viam/ur/module/utils.hpp

```diff
@@ -0,0 +1,52 @@
+#include <ur_client_library/log.h>
+
```

Comment:
You should include `<viam/sdk/log/logging.hpp>` and `<sstream>` explicitly. Right now they are included by resource.hpp but if that changes in the future then compilation would break.

Reply:
added